content/post/agile-analytics-p2.md

```diff
@@ -0,0 +1,56 @@
+
+Exploratory data analysis is an important part of the "basic research" function of an analytics or data science team. Unfortunately, it's really hard to write "user stories" or "acceptance criteria" for this sort of work because ... the whole point is that you aren't sure what you're going to find, and it requires a researcher's intuition to know when the story is "done" (because there's always more things you *could* look into, but often those marginal investigations aren't valuable).
+
+The nature of exploratory data analysis means that the objectives of the analysis may change *as you do the work*. You sometimes realize "oh x isn't actually very interesting -- the real issue is understanding y." If your scrum process is too inflexible, you can end up expending effort against tasks that aren't valuable simply because they're required to get the story done.  While (software) engineering is a discipline of creatively applying mathematical tools to a well-defined problem, data science is about creatively applying mathematical tools to *poorly defined* problems, and it's this fundumental difference in nature that makes partitioning the work into small well-scoped tasks sometimes impossible for data science.
+
+## Product ownership / story-writing
+
+When you're trying to build a model of customer churn, there's some amount of judgement and art that goes into doing good data science. The acceptance criteria of "predict which customers are going to churn" is too vague to be helpful and doesn't capture the art of investigating a problem by poking at it from multiple sides.  Additionally, it can be very difficult for someone external to write stories or be the product owner for the team. Often, the data scientists or analysts are *the* domain experts in the company and there isn't anyone else who is better informed about what the next step in a research program should be. 
+
+In user-facing product organizations, stories are often phrased like "as a customer trying to x, I should be able to y." This sort of framing often doesn't work well for research tasks since there isn't a customer your building a product for. Finally, bringing in an external product manager can disrupt the "flow" of research and prioritize software "features" over generating valuable intelligence.
+
+## Business-as-usual support
+
+In many analytics teams, there's a fair amount of business-as-usual reporting and analysis support. For example, the analytics team may be in charge of assembling the quarterly board deck, preparing reports for the monthly business reviews, or analyzing the results of ongoing A/B tests. Additionally, analytics teams need to be "on call" to support their business partners as questions come up ("how do I build this report?", "why doesn't this number match this other number?"). This sort of work falls outside of sprint planning, and if you don't account for it somehow, you'll end up consistently missing your sprint commitments.  
```

Comment:
This is a great example of how data teams have to both support internal data requests and needs, and also be responsible for delivering insights and data products. AFAIK the former doesn't fit the agile framework out of the box.